allensdk/brain_observatory/multi_stimulus_running_speed/multi_stimulus_running_speed.py

```diff
@@ -0,0 +1,330 @@
+import numpy as np
+import pandas as pd
+from allensdk.brain_observatory.sync_dataset import Dataset as SyncDataset
+from allensdk.brain_observatory import sync_utilities
+import argschema
+import json
+
+from allensdk.brain_observatory.multi_stimulus_running_speed._schemas import (
+    MultiStimulusRunningSpeedInputParameters,
+    MultiStimulusRunningSpeedOutputParameters
+)
+
+from allensdk.brain_observatory.behavior.data_objects.\
+    running_speed.running_processing import (
+        get_running_df
+    )
+
+
+class MultiStimulusRunningSpeed(argschema.ArgSchemaParser):
+    default_schema = MultiStimulusRunningSpeedInputParameters
+    default_output_schema = MultiStimulusRunningSpeedOutputParameters
+
+    START_FRAME = 0
+
+    def _extract_dx_info(
+        self,
+        frame_times: np.ndarray,
+        start_index: int,
+        end_index: int,
+        pkl_path: str
+    ) -> pd.core.frame.DataFrame:
+        """
+        Extract all of the running speed data
+
+        Parameters
+        ----------
+        frame_times: numpy.ndarray
+            list of the vsync times
+        start_index: int
+            Index to the first frame of the stimulus
+        end_index: int
+           Index to the last frame of the stimulus
+        pkl_path: string
+            Path to the stimulus pickle file
+
+        Returns
+        -------
+        pd.DataFrame
+
+        Notes
+        -------
+            velocity pd.DataFrame:
+                columns:
+                    "velocity": computed running speed
+                    "net_rotation": dx in radians
+                    "frame_indexes": frame indexes into
+                        the full vsync times list
+
+            raw data pd.DataFrame:
```

Comment:
Unsure what these notes refer to. Are these notes intended? There is still an indentation issue.